base/src/main/kotlin/com/freeletics/gradle/plugin/FreeleticsBaseExtension.kt

```diff
@@ -11,18 +10,13 @@ public abstract class FreeleticsBaseExtension(protected val project: Project) {
     fun explicitApi() {
         project.kotlin {
             explicitApi()
-
-            // TODO for Android projects the above isn't enough https://youtrack.jetbrains.com/issue/KT-37652
-            compilerOptions(project) {
-                freeCompilerArgs.add("-Xexplicit-api=strict")
-            }
         }
     }
 
-    fun optIn(vararg optIn: String) {
+    fun optIn(vararg classes: String) {
         project.kotlin {
-            compilerOptions(project) {
-                freeCompilerArgs.addAll(optIn.map { "-opt-in=$it" })
+            compilerOptions {
+                optIn.addAll(*classes)
             }
```

Comment:
Now uses the DSL version instead of adding raw compiler args https://kotlinlang.org/docs/whatsnew19.html#new-compiler-options